### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- #719
- [29.0.8](https://github.com/YunoHost-Apps/nextcloud_ynh/pull/735/commits/7a23074a5c2fd386915806052a23a609997f9da9)
- [Add FR translation for config panel](https://github.com/YunoHost-Apps/nextcloud_ynh/pull/735/commits/cc04eb60034cbf633190bd746d35a1d284486bac)
- #742
- #743
- #744

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by kay0u** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Review by kay0u — APPROVED** [review]

**Comment by kay0u** [comment]
We should merge #714 first, so ppl can use it to stay on version 29 when the version 30 is ready

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -4,10 +4,19 @@ source _common.sh
...
 
-phpflags="--define apc.enable_cli=1"
+#=================================================
+# INITIALIZE AND STORE SETTINGS
+#=================================================
 
+phpflags="--define apc.enable_cli=1"
 ynh_app_setting_set --app=$app --key=phpflags --value="$phpflags"
 
+enable_notify_push="0"
```
Comment (Salamandar) [inline scripts/install:14]: This is supposed to come from the manifest, right ??

**Comment by ericgaspar** [comment]
> We should merge https://github.com/YunoHost-Apps/nextcloud_ynh/pull/714 first, so ppl can use it to stay on version 29 when the version 30 is ready

@kay0u What do you think if we merge this PR and wait for the next minor release (29.0.9) to integrate the stable/oldstable option?

File: scripts/install (hunk 1 above)
Comment (kay0u) [reply scripts/install:14]: This is no longer the case, as we want to simplify the installation. The advanced settings should still be in the config panel.

File: tests.toml [hunk 2]
```diff
@@ -8,15 +8,38 @@ test_format = 1.0
...
 
-    test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
-    test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
-    test_upgrade_from.9c6d1eea.name = "Upgrade from 27.1.4"
-    test_upgrade_from.e9f82ab7.name = "Upgrade from 28.0.6"
-    test_upgrade_from.e9f82ab7.args.system_addressbook_exposed = "yes"
+    #test_upgrade_from.c5cf91ad.name = "Upgrade from 25.0.2"
+    #test_upgrade_from.caf917f3.name = "Upgrade from 26.0.2"
+    #test_upgrade_from.9c6d1eea.name = "Upgrade from 27.1.4"
+    #test_upgrade_from.e9f82ab7.name = "Upgrade from 28.0.6"
```
Comment (kay0u) [inline tests.toml:14]: Why did you disable all the upgrade tests?

IMO, we should add tests from "old" nextcloud versions, but the advanced settings (such as notify push) should not run a full tests set

**Review by kay0u — APPROVED** [review]
Yes, we can merge it and continue to iterate on the "oldstable" branch :+1:

File: tests.toml (hunk 2 above)
Comment (ericgaspar) [reply tests.toml:14]: The CI has a timeout that prevents all Nextcloud tests from being performed successfully... I don't know if we can add timeout exception for some applications in CI

**Pull request merged**